templates/keycloak.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Jboss Application Server
+After=network.target
+
+[Service]
+Type=idle
+Environment=JBOSS_HOME={{ keycloak_parent_install_dir }}/keycloak JBOSS_LOG_DIR={{ keycloak_log_dir }} "JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m"
+User=keycloak
+Group=keycloak
+ExecStart={{ keycloak_parent_install_dir }}/keycloak/bin/standalone.sh
+TimeoutStartSec=600
+TimeoutStopSec=600
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Its possible to use this role to configure the systemd service   https://github.com/onaio/ansible-role-service

Reply:
nice

Reply:
@githengi I can seem to find an example in the whole onaio repo. I see the collectd role has done same way I have done.

Reply:
https://github.com/onaio/ansible-role-tomcat

Reply:
nice, I see how to pass the command, still looking on how to pass the environment part

Reply:
environment= {{ "JBOSS_HOME={{ keycloak_parent_install_dir }}/keycloak JBOSS_LOG_DIR={{ keycloak_log_dir }} "JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m" }}

Reply:
This is not mentioned anywhere in the `ansible-role-service`. But I have discovered there is no need for this because `JAVA_OPTS` is already defined in the configs.